app/uploaders/avatar_uploader.rb

```diff
@@ -0,0 +1,52 @@
+class AvatarUploader < CarrierWave::Uploader::Base
+
+  # Include RMagick or MiniMagick support:
+  # include CarrierWave::RMagick
+  include CarrierWave::MiniMagick
+  version :thumb do
+  process :resize_to_fill => [50, 50]
+end
```

Comment:
Ah, sorry, I see now, it's a whitespace issue. For readability, I recommend indenting the contents of the `version` block:

```ruby
class AvatarUploader < CarrierWave::Uploader::Base
  # ...
  version :thumb do
    process resize_to_fill: [50, 50]
  end

  storage :file
  # etc.
end
```